packages/riverpod_analyzer_utils_tests/test/ref_invocation_test.dart

```diff
@@ -575,6 +645,123 @@ void fn(_Ref ref) {
     );
   });
 
+  testSource('Decodes nested ref.watch invocations with family providers',
```

Comment:
Those tests are redundant as it is.

But we'd need a test for WidgetRef instead of Ref

Reply:
I've removed these tests and added a test in the widget ref invocation file